src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+
+/**
+ * Represents a command to be executed by the chatbot. A <code>Command</code> object
+ * corresponds to an executable command.
+ */
+public abstract class Command {
```

Comment:
I like how you have created an abstract class Command in order to handle all existing commands to enhance code readability!